kernelci/runtime/lava.py

```diff
@@ -342,6 +342,8 @@ def _submit(self, job):
         resp = self._server.session.post(
             jobs_url, json=job_data, allow_redirects=False
         )
+        if resp.status_code >= 400:
```

Comment:
I'd suggest using a specific code i.e. for a successful response to use here to cover the rest of the codes.
I believe we receive HTTP 200 on success?
```suggestion
        if resp.status_code != 200:
```

Reply:
> It might be 201, 202, which are valid too. Error codes are starting from 400.

Yes, that's true. I am not aware of different HTTP codes returned by LAVA jobs, but I thought it might be just one code for successful submission.